github/resource_github_organization_ruleset.go

```diff
@@ -269,6 +269,51 @@ func resourceGithubOrganizationRuleset() *schema.Resource {
 							Optional:    true,
 							Description: "Prevent users with push access from force pushing to branches.",
 						},
+						"merge_queue": {
```

Comment:
merge queue is not supported on the org level, please see my PR description in: https://github.com/google/go-github/pull/3253 for details

Reply:
Removed